src/Matcher/Pattern/TypePattern.php

```diff
@@ -69,10 +78,28 @@ public function matchExpanders($value)
     }
 
     /**
-     * @return null|string
+     * {@inheritdoc}
      */
     public function getError()
     {
         return $this->error;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function hasExpander(string $expanderName)
+    {
+        foreach ($this->expanders as $expander) {
+            if (!$this->expanderInitializer->hasExpanderDefinition($expanderName)) {
```

Comment:
in this case, I propose to I add abstract `Expander` class with `is($name)`. What do you think about that? Are there any disadvantages to using abstract expander?